ui/src/components/Logout.js

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { useAuth } from "../auth/AuthProviderDefault";
+import { Icon, Message } from "semantic-ui-react";
+
+const Logout = () => {
+  const { sendRequestCommon } = useAuth();
+  const [errorMessage, setErrorMessage] = useState("");
+  const onSignOut = useCallback(async () => {
+    const signOutResponse = await sendRequestCommon(
+      null,
+      "/api/v2/logout",
+      "get"
+    );
+
+    if (!signOutResponse) {
+      return;
+    }
+
+    if (signOutResponse.redirect_url) {
+      window.location.assign(signOutResponse.redirect_url);
+    }
+
+    setErrorMessage(signOutResponse.message);
+  }, [sendRequestCommon]);
+
+  useEffect(() => {
```

Comment:
do we actually remove or update the user and isSessionExpired states? I think protectedRoute component will still allow the logged out person if those states are cleaned up. We will need to create functions in AuthProviderDefault.js to clean up those session related states.